components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/ApisApiServiceImpl.java

```diff
@@ -459,7 +459,7 @@ private API prepareToCreateAPIByDTO(APIDTO body) throws APIManagementException {
         assignLabelsToDTO(body,apiToAdd);
 
         // set default API Level Policy
-        if (StringUtils.isBlank(apiToAdd.getApiLevelPolicy())) {
+        if (apiToAdd.getApiLevelPolicy() != null) {
```

Comment:
```suggestion
        if (StringUtils.isEmpty(apiToAdd.getApiLevelPolicy())) {
```

How about this?

Reply:
I'm a bit confused. The difference between isBlank and isEmpty is isEmpty returns false for strings with only spaces and isBlank returns true in that case. So are we differentiate between { "apiLevelPolicy": "" .. } and  { "apiLevelPolicy": " " .. } ?

Reply:
@Meruja is this solved ?

Reply:
Yes, the entire logic has been changed with 05d94d28629d33b021fad9cf209a099633302d11